environment/php/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG PHP_VERSION=8.3.1-fpm-alpine
```

Comment:
Do we need fpm for this project? I think we can use `cli` image. 
Also I would like to switch to Debian 12 (bookworm) instead of Alpine which is more problematic in some cases.

Reply:
changed